README.md

```diff
@@ -70,25 +70,6 @@ Flags:
 Use "subnet-cli [command] --help" for more information about a command.
 ```
 
-#### Ledger Support
-
-To use your [Ledger](https://www.ledger.com) with `subnet-cli`, just add the
-`-l`/`--ledger` flag to any command below.
-
-For example, to create 4 node network on Fuji with Ledger, you would run:
-
-```bash
-subnet-cli wizard \
---ledger \
---node-ids=NodeID-741aqvs6R4iuHDyd1qT1NrFTmsgu78dc4,NodeID-K7Y79oAmBntAcdkyY1CLxCim8QuqcZbBp,NodeID-C3EY6u4v7DDi6YEbYf1wmXdvkEFXYuXNW,NodeID-AiLGeqQfh9gZY3Y8wLMD15tuJtsJHq5Qi \
---vm-genesis-path=fake-genesis.json \
---vm-id=tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH \
---chain-name=test
-```
-
-_Make sure you've downloaded the latest version of the
-[Avalanche Ledger App](https://docs.avax.network/learn/setup-your-ledger-nano-s-with-avalanche)!_
-
```

Comment:
why remove ledger support?

Reply:
It's made of `github.com/ava-labs/avalanche-ledger-go` sdk which seems to work only with Avalanche wallet.

Reply:
Ledger removed completely in 0b45dd25d647d0df62e00d82d638bc47cb20a4d8